lib/contentmanager.js

```diff
@@ -93,8 +105,8 @@ ContentPlugin.prototype.onDatabaseCreated = function (db, next) {
 
   fs.readFile(schemaPath, function (err, data) {
     if (err) {
-      logger.log('error', 'could not read schema file at ' + schemaPath);
-      return next(err);
+      logger.log('info', 'could not read schema file at ' + schemaPath);
+      return next(null);
```

Comment:
Should this log still be type error and should this return the error rather than null?

Reply:
If we intentionally return the error then it will revert to the previous functionality where none of the schema is loaded if one is missing.

Reply:
Ok :+1: I did wonder but wanted to check